op-chain-ops/script/script.go

```diff
@@ -436,8 +447,8 @@ func (h *Host) onEnter(depth int, typ byte, from common.Address, to common.Addre
 		return
 	}
 
-	// Bump nonce value, such that a broadcast Call appears like a tx
-	if parentCallFrame.LastOp == vm.CALL {
+	// Bump nonce value, such that a broadcast Call or CREATE2 appears like a tx
+	if parentCallFrame.LastOp == vm.CALL || parentCallFrame.LastOp == vm.CREATE2 {
```

Comment:
Good catch. Checked, and the create2 opcode does increment the nonce of the caller, event though create2 address computation does not depend on it. TIL.